data/blog/意大利罗马夏天四日游.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: 夏天意大利罗马四日游
 date: 2025-08-14T19:34:28.147Z
+external_discussions:
+  - platform: v2ex
+    url: https://v2ex.com/t/1158986
 ---
```

Comment:
This is nice, please ensure `app/editor/page.tsx` component have the input to add the `external_discussions` information as well.